music21/roman.py

```diff
@@ -628,105 +628,47 @@ def correctRNAlterationForMinor(figureTuple, keyObj):
     return FigureTuple(figureTuple.aboveBass, alter, rootAlterationString)
 
 
-def romanNumeralFromChord(chordObj,
+def romanNumeralFromChord(chordObj: chord.Chord,
                           keyObj: Union[key.Key, str] = None,
-                          preferSecondaryDominants=False):
+                          preferSecondaryDominants: bool = True):
     # noinspection PyShadowingNames
+    # TODO: preferSecondaryDominants option
     '''
-    Takes a chord object and returns an appropriate chord name.  If keyObj is
-    omitted, the root of the chord is considered the key (if the chord has a
-    major third, it's major; otherwise it's minor).  preferSecondaryDominants does not currently
-    do anything.
+    Takes a chord object (chordObj) and returns a Roman numeral.
+    As Roman numerals are key-relative, this function makes most sense when the
+    user explicitly provides a key context with the keyObj variable.
 
-    >>> rn = roman.romanNumeralFromChord(
-    ...     chord.Chord(['E-3', 'C4', 'G-6']),
-    ...     key.Key('g#'),
-    ...     )
-    >>> rn
-    <music21.roman.RomanNumeral bivo6 in g# minor>
-
-    The pitches remain the same with the same octaves:
-
-    >>> for p in rn.pitches:
-    ...     p
-    <music21.pitch.Pitch E-3>
-    <music21.pitch.Pitch C4>
-    <music21.pitch.Pitch G-6>
-
-    >>> romanNumeral2 = roman.romanNumeralFromChord(
-    ...     chord.Chord(['E3', 'C4', 'G4', 'B-4', 'E5', 'G5']),
-    ...     key.Key('F'),
-    ...     )
-    >>> romanNumeral2
-    <music21.roman.RomanNumeral V65 in F major>
-
-    Note that vi and vii in minor signifies what you might think of
-    alternatively as #vi and #vii:
```

Comment:
Is there a reason these docs were deleted?

Reply:
as with JTW